simexpal/launch/slurm.py

```diff
@@ -42,13 +42,16 @@ def _do_submit(self, cfg, experiment, runs):
 		use_array = len(locked) > 1
 
 		# Build the specfile.
+		manifest = common.compile_manifest(locked[0])
 		if not use_array:
 			specs = {
-				'manifest': common.compile_manifest(locked[0]).yml
+				'manifest': manifest.yml
 			}
 		else:
+			manifests = [manifest.yml]
+			manifests.extend([common.compile_manifest(run).yml for run in locked[1:]])
 			specs = {
-				'manifests': [common.compile_manifest(run).yml for run in locked]
+				'manifests': manifests
```

Comment:
This looks strange to me. If we're submitting multiple runs here, is it guaranteed that they all have the same variants? If not, the code below would break since it always extracts the `VARIANT_VALUE` of the first run.

Reply:
Yes, it is guaranteed that the runs have the same variants since we group them by their experiments in ``submit_multiple()`` and experiments are defined by their variations (among their revision and ``ExperimentInfo``-object).

Reply:
Maybe we should still verify this here.